engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-details.html

```diff
@@ -32,6 +32,22 @@ <h2 translate>{{ repository.name }}</h2>
           </span>
         </li>
 
+        <li role="menuitem">
+          <a ng-click="republishRepository(repository)"
+             ng-hide="syncInProgress(repository.last_sync) || denied('edit_products', product)"
+             translate>
+            Republish Repository Metadata
+          </a>
+
+          <span class="disabled" ng-show="syncInProgress(repository.last_sync)" translate>
+            Cannot republish Repository, a sync is already in progress.
+          </span>
+
+          <span class="disabled" ng-show="denied('sync_products', product)" translate>
+            Cannot republish Repository without the proper permissions.
+          </span>
+        </li>
```

Comment:
I modeled this after the sync button.  I can understand wanting a link to the action, but why would you want to have to click to get it?